tests/basic_test.rs

```diff
@@ -8,7 +8,7 @@ use std::env::temp_dir;
 
 #[test]
 fn test_in_single_thread() -> Result<()> {
-    let dir = temp_dir();
+    let dir = temp_dir().join("pancake");
```

Comment:
For some reason I can't operate directly on `/tmp`. Did we mean to `mktemp` here?

Reply:
:+1: